### PR TITLE
Support Rails 7.1.4

### PR DESCRIPTION
**Pull request by ascullion-FS**
In Rails 7.1.4.x, the `end`  property from the ActiveSupport::Notifications event is expressed in seconds since epoch instead of milliseconds since epoch. This causes the `job_latency` method to fail along with most of the enqueued jobs in our test suite. Fixing this issue is required to allow us to upgrade our version of Rails.

### CODE REVIEW

**Comment by andrewmarkle** [comment]
Looks great! @ascullion-FS can you:
 * Update the testing matrix to include Rails 7.1 and 7.2 here? https://github.com/Fullscript/yabeda-activejob/blob/main/.github/workflows/test.yml#L20
 * Update the changelog.

File: lib/yabeda/activejob/version.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module Yabeda
   module ActiveJob
-    VERSION = "0.5.0"
+    VERSION = "0.5.1"
```
Comment (andrewmarkle) [inline lib/yabeda/activejob/version.rb]: We'll have to bump this to 0.6.0 as I adjusted the minimum Ruby / Rails version in the previous PR.

**Comment by andrewmarkle** [comment]
I went through the open PRs and saw that Rosa had opened one to fix this first so I merged that one in. https://github.com/Fullscript/yabeda-activejob/pull/12

Let's keep your specs though and work on a release.

**Comment by ascullion-FS** [comment]
> I went through the open PRs and saw that Rosa had opened one to fix this first so I merged that one in. #12
> 
> Let's keep your specs though and work on a release.

Rosa's fix doesn't support version 7.1.3, though, as far as I understand. I'll see if I can combine them because I prefer her syntax.

**Comment by ascullion-FS** [comment]
I'm going to go test the new code against our repo.

**Comment by ascullion-FS** [comment]
> Looks great! @ascullion-FS can you:
> 
> * Update the testing matrix to include Rails 7.1 and 7.2 here? https://github.com/Fullscript/yabeda-activejob/blob/main/.github/workflows/test.yml#L20
> * Update the changelog.

I added Rails 7.2 but had to remove Ruby 3.0 because it's not compatible with Rails 7.2

**Review by andrewmarkle — APPROVED** [review]
❤️ Yay! Thank you.

**Pull request merged**